builtin/game/chatcommands.lua

```diff
@@ -71,7 +71,7 @@ end
 --
 core.register_chatcommand("me", {
 	params = "<action>",
-	description = "chat action (eg. /me orders a pizza)",
+	description = "Display chat action (eg. /me orders a pizza)",
```

Comment:
Still seems unclear what this does (although most know already).
"Add player name to the start of the entered text" or something?